kickstarts/partials/post/db_init.ks.erb

```diff
@@ -0,0 +1,27 @@
+# Create a script to initialize appliance on first boot
+cat > /bin/appliance-initialize.sh <<EOF
+#!/bin/sh
+[[ -d /var/opt/rh/rh-postgresql94/lib/pgsql/data/base ]] && exit 0
+[[ -s /etc/default/evm ]] && source /etc/default/evm
+echo "Initializing Appliance, please wait ..." > /dev/tty1
+appliance_console_cli --region 0 --internal --password smartvm
+EOF
```

Comment:
Yes, it's nice that it's separated out because it's easier to grok all by itself.